keel-sql/keel-sql.gradle.kts

```diff
@@ -65,16 +67,37 @@ dependencies {
   liquibaseRuntime("mysql:mysql-connector-java")
 }
 
-tasks.register<Exec>("runMysql") {
-  commandLine("docker run --name mysqlJooq -d --rm -e MYSQL_ROOT_PASSWORD=sa -e MYSQL_DATABASE=keel -p 6603:3306 mysql:5.7".split(" "))
-  doLast {
-    // Wait for the DB server to come up...
-    sleep(15 * 1000)
-  }
+tasks.register<Exec>("startMysql") {
+  commandLine("sh", "-c",
+    "docker stop mysqlJooq || " +
+      "docker run --name mysqlJooq -d --rm -e MYSQL_ROOT_PASSWORD=sa -e MYSQL_DATABASE=keel -p 6603:3306 mysql:5.7 && " +
+      "sleep 15"
+  )
+  enabled = false
+  dependsOn("mysqlSwitch")
 }
 
 tasks.register<Exec>("stopMysql") {
   commandLine("docker stop mysqlJooq".split(" "))
+  enabled = false
+  dependsOn("mysqlSwitch")
+}
+
+tasks.register("mysqlSwitch") {
```

Comment:
This was the only trick I could find to make this work. This task has the same caching config as `jooqGenerate`, which means it will only run if there's changes to relevant jooq files, and when it does, it enables the start/stop tasks to run MySQL.